src/allencell_ml_segmenter/postprocess/postprocess_view.py

```diff
@@ -0,0 +1,221 @@
+        threshold_group_layout.addLayout(specific_value_layout)
+
+        # autothresholding
+        autothreshold_layout = QHBoxLayout()
+        self._autothreshold_radio_button: QRadioButton = QRadioButton()
+        auto_thresh_label: LabelWithHint = LabelWithHint("Autothreshold")
+        auto_thresh_label.set_hint("Apply an autothresholding method.")
+
+        self._autothreshold_method_combo: QComboBox = QComboBox()
+        self._autothreshold_method_combo.addItems(["Otsu"])
+        self._autothreshold_method_combo.setEnabled(False)
+
+        autothreshold_layout.addWidget(self._autothreshold_radio_button)
+        autothreshold_layout.addWidget(auto_thresh_label)
+        autothreshold_layout.addWidget(self._autothreshold_method_combo)
+        threshold_group_layout.addLayout(autothreshold_layout)
+
+        threshold_group_box.setLayout(threshold_group_layout)
+        layout.addWidget(threshold_group_box)
+
+        # apply and save
+        self._apply_save_button: QPushButton = QPushButton("Apply & Save")
+        self._apply_save_button.setEnabled(False)
+        layout.addWidget(self._apply_save_button)
+
+        # need styling
+        self.setStyleSheet(Style.get_stylesheet("thresholding_view.qss"))
+
+        # configure widget behavior
+        self._configure_slots()
+
+    def _configure_slots(self) -> None:
+        """
+        Connects behavior for widgets
+        """
+
+        # enable selections when corresponding radio button is selected
+        self._specific_value_radio_button.toggled.connect(
+            lambda checked: self._enable_specific_threshold_widgets(checked)
+        )
+        self._autothreshold_radio_button.toggled.connect(
+            lambda checked: self._autothreshold_method_combo.setEnabled(
+                checked
+            )
+        )
+
+        # enable apply button only when thresholding method is selected
+        self._none_radio_button.toggled.connect(self._enable_apply_button)
+        self._specific_value_radio_button.toggled.connect(
+            self._enable_apply_button
+        )
+        self._autothreshold_radio_button.toggled.connect(
+            self._enable_apply_button
+        )
+
+    def _update_spinbox_from_slider(self, value: int) -> None:
+        """
+        Update the spinbox value when slider is changed
+        """
+        self._threshold_value_spinbox.setValue(value / 100.0)
+
+    def _update_slider_from_spinbox(self, value: float) -> None:
+        """
+        Update the slider value when spinbox is changed
+        """
+        self._threshold_value_slider.setValue(int(value * 100))
+
+    def _enable_specific_threshold_widgets(self, enabled: bool) -> None:
+        """
+        enable or disable specific value thresholding widgets
+        """
+        self._threshold_value_slider.setEnabled(enabled)
+        self._threshold_value_spinbox.setEnabled(enabled)
+
+    def _enable_apply_button(self) -> None:
```

Comment:
This is a new widget, which doesn't yet perform thresholding right?  (i see no calls to a model)

Reply:
correct. purely UI changes here